src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -80,13 +83,79 @@ def _print_extensions_msg(help_file):
             if help_file.command_source.preview:
                 logger.warning(help_file.command_source.get_preview_warn_msg())
 
+    # TODO: should this be _print
```

Comment:
Reason?

Reply:
The method is not used. 

I will remove the parser module.